src/test/java/com/qunite/api/web/UserControllerIntegrationTest.java

```diff
@@ -63,6 +60,7 @@ public void cleanAll() {
 
 
   @Test
+  @WithMockUser("First")
```

Comment:
same as in QueueControllerIntegrationTest.java

Reply:
same as in QueueControllerIntegrationTest.java